lms/static/sass/application.scss.mako

```diff
@@ -28,72 +28,7 @@
   @import '${env.get('THEME_NAME')}';
 % endif
 
-@import 'base/base';
-
-// base - assets
-@import 'base/font_face';
-@import 'base/extends';
-@import 'base/animations';
-
-// base - starter
-@import 'base/base';
-
-// base - elements
-@import 'elements/typography';
-@import 'elements/controls';
-@import 'elements/system-feedback';
-
-// shared - course
-@import 'shared/fields';
-@import 'shared/forms';
-@import 'shared/footer';
-@import 'shared/header';
-@import 'shared/course_object';
-@import 'shared/course_filter';
-@import 'shared/modal';
-@import 'shared/activation_messages';
-@import 'shared/unsubscribe';
-
-// shared - platform
-@import 'multicourse/home';
-@import 'multicourse/dashboard';
-@import 'multicourse/account';
-@import 'multicourse/courses';
-@import 'multicourse/course_about';
-@import 'multicourse/jobs';
-@import 'multicourse/media-kit';
-@import 'multicourse/about_pages';
-@import 'multicourse/press_release';
-@import 'multicourse/error-pages';
-@import 'multicourse/help';
-@import 'multicourse/edge';
-@import 'multicourse/survey-page';
-
-// base - specific views
-@import "views/account-settings";
-@import "views/learner-profile";
-@import 'views/login-register';
-@import 'views/verification';
-@import 'views/decoupled-verification';
-@import 'views/shoppingcart';
-@import 'views/homepage';
-@import 'course/auto-cert';
-
-// applications
-@import "discussion/utilities/variables";
-@import "discussion/mixins";
-@import 'discussion/discussion'; // Process old file after definitions but before everything else
-@import "discussion/elements/actions";
-@import "discussion/elements/editor";
-@import "discussion/elements/labels";
-@import "discussion/elements/navigation";
-@import "discussion/views/thread";
-@import "discussion/views/create-edit-post";
-@import "discussion/views/response";
-@import 'discussion/utilities/developer';
-@import 'discussion/utilities/shame';
-
-@import 'news';
+@import 'build-lms'; // shared app style assets/rendering
 
 ## Import styles for search
 % if env["FEATURES"].get("ENABLE_DASHBOARD_SEARCH", False):
```

Comment:
Hmm... Hadn't seen this before. I know it's not your commit, but it might be worth digging in at some point to see if its necessary to exclude this Sass import. Usually it doesn't hurt much to include it, and it's one less non-Sass/CSS thing in our import...